src/components/global/Header.js

```diff
@@ -0,0 +1,44 @@
+import React from 'react';
+
+export default function Header(){
```

Comment:
May be easier to add the router links here instead of in the app component, unless you plan to reuse this elsewhere.